services/QuillLMS/client/app/bundles/PremiumHub/components/usage_snapshots/snapshotCount.tsx

```diff
@@ -32,6 +32,7 @@ interface SnapshotCountProps {
 
 const PUSHER_CURRENT_EVENT_KEY = 'admin-snapshot-count-cached'
 const PUSHER_PREVIOUS_EVENT_KEY = 'admin-snapshot-previous-count-cached'
+const NOT_APPLICABLE = 'N/A'
```

Comment:
Excellent call making this a constant.  I probably should have done that.

Reply:
just wanted to flag that we actually already have this via the Shared bundle!